packages/common/src/intl/locales/en/replenishment.json

```
@@ -2,35 +2,37 @@
   "button.delete-lines": "Delete lines",
   "button.new-shipment": "New Shipment",
   "button.save-and-confirm-status": "Confirm {{status}}",
-  "dev.log-draft": "Log draft",
   "error.requisition-not-found": "Requisition not found",
   "error.shipment-not-found": "Shipment not found",
+  "heading.are-you-sure": "Are you sure?",
   "heading.related-documents": "Related documents",
   "label.add-batch": "Add batch",
+  "label.draft": "Draft",
+  "label.finalised": "Finalised",
   "label.group-by-item": "Group by Item",
   "label.location": "Location",
+  "label.new": "New",
+  "label.order-quantity": "Order quantity",
   "label.pricing": "Pricing",
   "label.quantities": "Quantities",
   "label.select-rows-to-delete-them": "Select rows to delete them",
+  "label.sent": "Sent",
+  "label.stock-details": "Stock details",
+  "label.suggested-quantity": "Suggested quantity",
   "label.supplier-name": "Supplier Name",
   "label.supplier-ref": "Supplier Ref",
+  "label.min-months-of-stock": "Min. Months of Stock",
```

```suggestion
  "label.min-months-of-stock": "Minimum MOS",
```

or "Minimum Months of Stock" ??
I think that we have enough space to have "Min." in full and that "MOS" is pretty well understood as an acronym, but perhaps "Min." is more english speakers only

---

I originally had minimum MOS but then changed as I though min. was more commonly used than MOS - but you're right, in the context it is very highly used